flaps/flaps.go

```diff
@@ -595,6 +596,7 @@ func (f *Client) sendRequest(ctx context.Context, method, endpoint string, in, o
 		return err
 	}
 	req.Header.Set("User-Agent", f.userAgent)
+	req.Header.Set("Invocation-ID", invocationID)
```

Comment:
Should we prefix this with `Flyctl-Invocation-ID`

Reply:
Yea that makes sense